### PR TITLE
chore: update examples for hashicorp/tls v4

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
examples/cloudrun/main.tf
--- old/examples/cloudrun/main.tf
+++ new/examples/cloudrun/main.tf
@@ -31,7 +31,7 @@
   ssl                             = var.ssl
   managed_ssl_certificate_domains = [var.domain]
   https_redirect                  = var.ssl
-  labels                          = {"Example Label"="Cloudrun Example"}
+  labels                          = { "Example Label" = "Cloudrun Example" }
 
   backends = {
     default = {
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

File: examples/cloudrun/main.tf [hunk 1]
```diff
@@ -24,14 +24,14 @@ provider "google-beta" {
 
 # [START cloudloadbalancing_ext_http_cloudrun]
 module "lb-http" {
-  source  = "GoogleCloudPlatform/lb-http/google//modules/serverless_negs"
-  version = "~> 6.2.0"
```
Comment (bharathkkb) [inline examples/cloudrun/main.tf:28]: So we have explicit versions here for surfacing in c.g.c. Is this giving an error otherwise?

Comment (apeabody) [reply examples/cloudrun/main.tf:28]: Hi @bharathkkb Yes, as 6.2.0 doesn't support lables.

**Review by bharathkkb — APPROVED** [review]

File: examples/cloudrun/main.tf (hunk 1 above)
Comment (apeabody) [reply examples/cloudrun/main.tf]: As discussed we'll update the example post-release that support labels. #262

**Pull request merged**